### ISSUE TITLE
bug: 500 Error when using non-existing attribute in `uniqueness_constraints`

### ISSUE THREAD

**Issue by BeArchiTek** [opsmill/infrahub#4677]
### Component

infrahubctl

### Infrahub version

0.16.3

### Current Behavior

When defining a schema with a node that includes a`uniqueness_constraints` field referencing a non-existing attribute, the system throws an internal server error (500).

Ideally, the system should provide a validation error or a clear message explaining that the referenced attribute does not exist.

### Expected Behavior

The system should validate the schema and return a descriptive error message

### Steps to Reproduce

Using the schema below with a`uniqueness_constraints`, referencing an attribute that does not exist in the node's attributes list.

```yaml
version: '1.0'

nodes:
  - name: Test
    namespace: Random
    attributes:
      - name: name
        kind: Text
    uniqueness_constraints:
      - ["non_existing_attribute__value"]
```

Load the schema.

The server returns a 500 error.
```console
> infrahubctl schema load test-schema.yml
HTTP communication failure: Server error '500 Internal Server Error' for url
'http://localhost:8000/api/schema/load?branch=main'
For more information check:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500 on POST to
http://localhost:8000/api/schema/load?branch=main
```

**Comment by ogenstad** [comment]
This looks like a backend bug.

**Comment by ogenstad** [comment]
Fixed in #4690.
